test/api/controllers/validators/resolvers/ResolversSpec.scala

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.controllers.validators.resolvers
+
+import api.models.errors.MtdError
+import cats.data.Validated.{Invalid, Valid}
+import cats.implicits.catsSyntaxOption
+import support.UnitSpec
+
+class ResolversSpec extends UnitSpec with Resolvers {
+  private val notInteger = MtdError("NOT_INT", "Not integer", 400)
+  private val outOfRange = MtdError("OUT_OF_RANGE", "Out of range", 400)
+  private val odd        = MtdError("ODD", "Odd", 400)
```

Comment:
Could we rename these so that they are clearly errors? I think it will make it easier to follow the tests and how the different resolvers/validators are being used/insantiated

```suggestion
  private val notIntegerError = MtdError("NOT_INT", "Not integer", 400)
  private val outOfRangeError = MtdError("OUT_OF_RANGE", "Out of range", 400)
  private val oddError        = MtdError("ODD", "Odd", 400)
```